osmpbfreader.h

```diff
@@ -190,9 +190,9 @@ struct Parser {
 
             // check that raw_size is set correctly
             if(sz != blob.raw_size())
-                warn() << "  reports wrong raw_size: " << blob.raw_size() << " bytes";
+                warn() << "  reports wrong raw_size: " << blob.raw_size() << " bytes";			
+                memcpy(unpack_buffer, blob.raw().c_str(), sz);
```

Comment:
This line is badly indented.

Reply:
Hi @TeXitoi .Very thanks to your review. To be truth, I am a very newbie in git. I think we can save our time by just close this PR, and you can edit the code yourself.